Formula/hsd.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Hsd < Formula
+  desc "Handshake Daemon & Full Node"
+  homepage "https://handshake.org"
+  url "https://github.com/handshake-org/hsd/archive/v2.1.2.tar.gz"
+  sha256 "4fc39ee5ca2b91186847a31b3244ae5d57235720ac40812cf752bb687c0833cf"
+
+  depends_on "python" => :build
+  depends_on "node@10"
+  depends_on "unbound"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
```

Comment:
Can you make this look explicitly for the node@10 npm using `#{Formula['node@10']}.bin/"npm"`

Reply:
Good call, this has been updated.